@ecomplus/storefront-template/template/js/lib/menu.js

```diff
@@ -1,9 +1,55 @@
-import { animateCss, $ } from '@ecomplus/storefront-twbs'
+import { isMobile, animateCss, $ } from '@ecomplus/storefront-twbs'
 import overlay from './overlay'
 
 const $menu = $('#menu')[0]
 let isVisible = false
 
+let $openSubMenu
+
+const closeSubmenu = () => {
+  $openSubMenu.style.display = 'none'
+  $openSubMenu = null
+}
+
+const toggleSubmenu = (slug, isClick) => {
+  if (!isMobile) {
+    if (isClick && slug) {
+      window.location = `/${slug}`
+    }
```

Comment:
```suggestion
  if (isClick && slug) {
    window.location = `/${slug}`
    return
  }
  if (!isMobile) {
```

geralmente em mobile isso não vai acontecer pq os links não serão visíveis, mas se por acaso acontecer o clique devee redirecionar também...